hoot-core/src/main/cpp/hoot/core/algorithms/linearreference/WaySublineMatch.cpp

```diff
@@ -37,18 +37,18 @@ WaySublineMatch::WaySublineMatch()
 }
 
 WaySublineMatch::WaySublineMatch(const WaySublineMatch& other, const ConstOsmMapPtr& newMap)
+  : _ws1(WaySubline(other.getSubline1(), newMap)),
+    _ws2(WaySubline(other.getSubline2(), newMap)),
+    _reversed(other.isReverseMatch())
 {
-  _ws1 = WaySubline(other.getSubline1(), newMap);
-  _ws2 = WaySubline(other.getSubline2(), newMap);
-  _reversed = other.isReverseMatch();
```

Comment:
Moved member variable initialization from the body of the constructor to the initializer list.